ape_etherscan/verify.py

```diff
@@ -265,10 +265,12 @@ def attempt_verification(self):
             standard_json = json.dumps(standard_input_json, indent=2)
             logger.debug(f"{standard_json}\n")
 
+        # if there is a folder structure for source_id like xx/yy/zzz.sol
+        request_source_id = source_id.split("/")[-1]
```

Comment:
potential OS slash bug?

```suggestion
        request_source_id = Path(source_id).as_posix().split("/")[-1]
```